adafruit_ble/advertising/standard.py

```diff
@@ -146,8 +138,9 @@ def __get__(self, obj, cls):
             obj._service_lists[first_adt] = BoundServiceList(obj, **self.__dict__)
         return obj._service_lists[first_adt]
 
-class ProvideServiceAdvertisement(Advertisement):
+class ProvideServicesAdvertisement(Advertisement):
```

Comment:
Trying to decide on the naming here. The old names (made plural) were 1) `ProvideServicesAdvertisement` and `SolicitServicesAdvertisement`. Those are parallel in consturction. Now it's 2) `ProvideServicesAdvertisement` and `ServicesSolicitationAdvertisement`, which aren't parallel.

We could go back to 1) or do 4) `ServicesAdvertisement` and `ServicesSolicitationAdvertisement`. 1) and 4) are both fine with me.

Reply:
Swapped back to 1.